app/Notifications/VerifyEmail.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Pterodactyl\Notifications;
+
+use Pterodactyl\Models\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class VerifyEmail extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    public User $user;
+    public string $name;
+    public string $token;
+
+    public function __construct(User $user, string $name, string $token)
+    {
+        $this->user = $user;
+        $this->name = $name;
+        $this->token = $token;
+    }
```

Comment:
Just make this:

```
public function __construct(private User $user, private string $name, private string $token)
...
```
By doing this, you do not need to define the types at the top, so you could get rid of:
```
   public User $user;
    public string $name;
    public string $token;
            $this->user = $user;
        $this->name = $name;
        $this->token = $token;
```
Make sure to test this! I am not 100% sure, but I believe this can be optimized.